src/utils/RegHubUtil.js

```diff
@@ -117,59 +121,97 @@ module.exports = {
     repositoryServerActions.reposLoading({repos: []});
 
     hubUtil.request({
-      url: `${REGHUB2_ENDPOINT}/namespaces/`,
+      url: `${REGHUB2_ENDPOINT}/namespaces/`
```

Comment:
I don't think we need to hit `/namespaces` anymore. Instead (like below), just getting the orgs (and append the user's name) is fine.